linera-chain/src/certificate/hashed.rs

```diff
@@ -66,3 +100,8 @@ impl<T> PartialEq for Hashed<T> {
 
 #[cfg(with_testing)]
 impl<T> Eq for Hashed<T> {}
+
+//TODO: Move
+pub trait Has<T> {
+    fn get(&self) -> &T;
+}
```

Comment:
Is it useful to be that general? Would a `HasChainId` trait be more readable, since its method could simply be called `chain_id`?

Reply:
Let's leave for now and if it turns out that there's only one version of it - I'll replace it.

Reply:
Please remove the TODO or add a task number

Reply:
Feeling a little bit like @afck here: I would probably choose the very explicit `HasChainId` (which can use the fact that `ChainId` is `Copy`) and otherwise why not `AsRef`.

Reply:
Soon we will need more instances for things like `Block` and `Epoch` (next PR will try to remove the `CertificateValue` which has all of these methods defined on the enum).